### ISSUE TITLE
Comment in resnet52_market.m

### ISSUE THREAD

**Issue by ahangchen** [layumi/Pedestrian_Alignment#4]
In [resnet52_market](https://github.com/layumi/Pedestrian_Alignment/blob/master/resnet52_market.m), [fc ans softmax layers](https://github.com/layumi/Pedestrian_Alignment/blob/master/resnet52_market.m#L20) are commented. 

When running [train_id_net_res_market_new](https://github.com/layumi/Pedestrian_Alignment/blob/master/train_id_net_res_market_new.m), missing 'label' layer will cause error, so it's needed to uncomment this lines when train base branch.

When running [train_id_net_res_market_align](https://github.com/layumi/Pedestrian_Alignment/blob/master/train_id_net_res_market_align.m), this code block should be commented, otherwise another error will be thrown out saying layer 'local_fc751's der is empty.  But commenting this block also causes an error saying that 'label_local' layer is missed.

Hope for a better instruction for this code.

**Comment by layumi** [comment]
Hi,
You're right. I add more illustration in the README.
Do the problem still keep appear? 
When training the basic network, we need the loss and dropout layers.
When training the whole network, we need to remove the loss in the 
[resnet52_market.m](https://github.com/layumi/Pedestrian_Alignment/blob/master/resnet52_market.m). We just want the net structure and params from ImageNet.

**Comment by ahangchen** [comment]
In net2, we need top1 loss for label_local, in net3, we don't need the losses.  If that's right, there exists a conflict when training the align net containing net2 and net3. My solution is commenting out the loss in [resnet52_market](https://github.com/layumi/Pedestrian_Alignment/blob/master/resnet52_market.m), and adding the loss layer additionally only for net2 in [train_id_net_res_market_align](https://github.com/layumi/Pedestrian_Alignment/blob/master/train_id_net_res_market_align.m), then the training run successfully, but this is different from the origin code so I am still confused about the correctness of the solution.
